gunicorn.conf.py

```diff
@@ -1,6 +1,6 @@
 import os
 
-bind = f'0.0.0.0:{os.getenv("PAPERLESS_PORT", 8000)}'
+bind = f'[::]:{os.getenv("PAPERLESS_PORT", 8000)}'
```

Comment:
Based on what I find for the documentation of [bind](https://docs.gunicorn.org/en/stable/settings.html#bind), should this be a list, with `[::]:8000` as one option?  I'm concerned this would enable only IPv6 as a is.

Reply:
`[::]` = Anything, `[::1]` = IPv6 localhost

After applying my change `ss -lnep` prints the following:

```
tcp                                LISTEN                              0                                   2048                                                                         *:8000                                                                      *:*                                 uid:911 ino:31050 sk:200e v6only:0 <->
``` 

As you can see v6only is 0. I've also included the curl commands for testing above. The curls were done inside the container. I've just checked and by installing paperless-ng on another machine and I was able to reach it over IPv4 and IPv6.

Reply:
Ok, great!